virtcontainers/container.go

```diff
@@ -1488,27 +1498,32 @@ func (c *Container) deleteCgroups() error {
 }
 
 func (c *Container) updateCgroups(resources specs.LinuxResources) error {
-	cgroup, err := cgroupsLoadFunc(cgroups.V1,
-		cgroups.StaticPath(c.state.CgroupPath))
-	if err != nil {
-		return fmt.Errorf("Could not load cgroup %v: %v", c.state.CgroupPath, err)
-	}
-
 	// Issue: https://github.com/kata-containers/runtime/issues/168
 	r := specs.LinuxResources{
 		CPU: validCPUResources(resources.CPU),
 	}
 
-	// update cgroup
-	if err := cgroup.Update(&r); err != nil {
-		return fmt.Errorf("Could not update cgroup %v: %v", c.state.CgroupPath, err)
-	}
-
 	// store new resources
 	c.config.Resources = r
 	if err := c.storeContainer(); err != nil {
 		return err
 	}
 
+	if c.sandbox.config.SandboxCgroupOnly {
```

Comment:
should this clause be at top of function?

Reply:
Added a comment why is needed here and not in top of function.

Reply:
Got it -- makes sense!